packages/cli/tsconfig.json

```diff
@@ -3,7 +3,7 @@
   "compilerOptions": {
     "baseUrl": ".",
     "module": "NodeNext",
-    "moduleResolution": "NodeNext",
+    "moduleResolution": "nodenext",
```

Comment:
VS Code would consistently autocomplete to `NodeNext` before, but today it is consistently recommending `nodenext`. Stylistic change from TS itself? Dunno.